### ISSUE TITLE
Google Cloud SQL Connectors

### ISSUE THREAD

**Issue by bsanderrmg** [airbytehq/airbyte#4114]
## Tell us about the new connector you’d like to have
Which source and which destination?

* Source - Google Cloud SQL (or update MySQL source connectors with client certificate validation)

Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 

* I don't believe so. I think the current version of the JDBC should handle this just fine, but would need a way to store the provided certificates in a keystore, or reference an existing keystore file.

## Describe the context around this new connector
Which team in your company wants this integration, what for? This helps us understand the use case.

* DevOps and technical non-IT users who create reports, who only have access to our data warehouse in BigQuery.

How often do you want to run syncs?

* Same intervals as MySQL native connector.

If this is an API source connector, which entities/endpoints do you need supported?

* N/A

## Describe the alternative you are considering or using

Currently the ways we've accomplished this is either via a Linked Server in SQL Server (via ODBC) or by pulling the rows down as a CSV and using the File connector to ingest.

**Comment by marcosmarxm** [comment]
@bsanderrmg we already support MySQL connector. Do you need to connect using SSL with client/server handshake?

**Comment by bsanderrmg** [comment]
@marcosmarxm No, Cloud SQL requires certificate authentication alongside a username/password. The fields highlighted below in MySQL Workbench reference the fields that I'm unable to provide using the MySQL connector in Airbyte.

Would be happy to be wrong, but I believe the connector as-is does not provide options for this.

![image](https://user-images.githubusercontent.com/53356092/121964673-e55f7380-cd39-11eb-8f98-376bb8a912d2.png)
